.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.8
+          python-version: 3.11
```

Comment:
If 3.10 is the documented minimum, then please stick to that so issues regarding language features used are caught by CI.